src/Maths/test/factorial.t.sol

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.13;
+pragma solidity ^0.8.0;
```

Comment:
Is there a reason why are we using an older version now?


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
I thought it would be a good idea to use ^0.8.0 so everyone can have a common base version to use.
The pragma version being ^0.8.0 does not mean it's an older version, It means it'll use version 0.8.0 and above.  

0.8 is the version most developers use now, I'm trying to prevent a situation where every dev who tries to commit to the repo uses a different pragma version:

Contributor 1 could use ^0.8.13 
Contributor 2 could use ^0.8.14 etc 

If everyone used ^0.8.0 it would serve as a common base version.